flake.nix

```diff
@@ -83,7 +83,9 @@
         haskellOverlay = self: super:
           builtins.listToAttrs (builtins.map ({ name, path }: {
             inherit name;
-            value = self.callCabal2nix name (./. + "/${path}") { };
+            value =
+              let p = self.callCabal2nix name (./. + "/${path}") { };
+              in haskellLib.appendConfigureFlag p "--ghc-options=-Werror";
```

Comment:
Does this also need `-Wall`? We have it in all the cabal files.

Reply:
This should already run with all the flags from the cabal files, we just need to add `-Werror` because we explicitly don’t want that in the cabal files.